alpine-rails-nginx/Dockerfile

```diff
@@ -19,24 +21,20 @@ RUN adduser -DSH nginx \
 
 # Create system directories and service symlinks
 RUN setup-directories.sh  root  r  /etc/service/nginx /etc/service/rails \
-  && setup-directories.sh nginx r  /etc/nginx \
   && setup-directories.sh nginx rw /var/run/nginx /var/cache/nginx /var/log/nginx \
-  && setup-directories.sh rails rw /opt/rails /var/run/rails \
-  && ln -s -- /etc/sv/nginx /etc/service/nginx/run \
-  && ln -s -- /etc/sv/rails /etc/service/rails/run
+  && setup-directories.sh rails r  /opt/rails \
+  && setup-directories.sh rails rw /var/run/rails /opt/rails/log /opt/rails/tmp \
+  && ln -s /etc/sv/nginx /etc/service/nginx/run \
+  && ln -s /etc/sv/rails /etc/service/rails/run \
+  && chown rails: /usr/bin/bundle
```

Comment:
Why this is required? Having non root owned files in `/usr` is IMO a bit odd, because none would ever expect this.